config/routes.rb

```diff
@@ -38,6 +38,13 @@
     end
   end
 
+  resources :collections do
+    member do
+      get :manifest, defaults: { format: :json }, as: "manifest_curation_concerns"
+      get :manifest, defaults: { format: :json }
```

Comment:
Nah, you can't, but I should probably find a solution to there being two helpers.